### ISSUE TITLE
Tests fail with CP problems with gradle-intellij-plugin 1.8.0

### ISSUE THREAD

**Issue by jshiell** [jshiell/checkstyle-idea#585]
Current guess is [source sets being added to the test classpath](https://github.com/JetBrains/gradle-intellij-plugin/commit/9530b23838b98dcab9cd854ecea1ef9ec20193a8).

[Release notes](https://github.com/JetBrains/gradle-intellij-plugin/releases/tag/v1.8.0)

**Comment by jshiell** [comment]
https://github.com/JetBrains/gradle-intellij-plugin/issues/1081

**Comment by jshiell** [comment]
Also true of 1.9 and 1.10.

**Comment by hsz** [comment]
Can you please check it with `1.12.3-SNAPSHOT`?

**Comment by jshiell** [comment]
I've just tried with `1.13.0`, which I presume supersedes `1.12.3-SNAPSHOT`, and I'm afraid it fails in the same manner.

**Comment by jshiell** [comment]
For completeness, I've tried with `1.12.3-SNAPSHOT` and it seems to blow up before that point with a strong dislike for aarch64. `runCsaccessTests` works with `1.12.0` and `1.13.0`.

```
> Task :runCsaccessTests FAILED
ProductInfo:
launch set = [null]
os.arch = aarch64
contains = false
listA = [null]
listB = [null]
fallback to aarch64

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':runCsaccessTests'.
> Could not find launch information for the current OS: Mac OS X (aarch64)
```

**Comment by hsz** [comment]
Ok, I will investigate it next week. Should I use the main branch or something else?
A brief repro scenario would be welcome.

**Comment by jshiell** [comment]
I haven't a simplified scenario at present, but it's easy to reproduce on this repo on `main`:

```
git clone git@github.com:jshiell/checkstyle-idea.git test
cd test
sed -i '' 's%1\.7\.0%1\.13\.0%g' build.gradle # or anything >= 1.8.0
./gradlew test
```

**Comment by jshiell** [comment]
Fixed in 1.13.3.
